test/lib/ufe/testDuplicateCmd.py

```diff
@@ -18,11 +18,14 @@
 
 import maya.cmds as cmds
 
-import usdUtils, mayaUtils
+import usdUtils
+import mayaUtils
```

Comment:
A bunch of these changes are because of the clang format now running.  I think it was the plan to just update files as we go?  I'm running the clang file that is checked into the repo so I'm assuming that we do want these edits.